tests/templates/forgot/text.handlebars

```diff
@@ -0,0 +1,17 @@
+Unified Listing Password Reset
+#######
+
+Your email address is associated with an account in the Common Terms
```

Comment:
This is a test template.  You'll also note that the heading is for one system, and the body is for another... ;)